### PR TITLE
Remove a potentially pre-existing server.pid in entrypoint.sh

### PR DESCRIPTION
**Pull request by raoulkramer**
This command will tigger the script to remove a potentially pre-existing server.pid for Rails. With this change the ops does not have to manually `rm tmp/pids/server.pid` remove the server.pid file.

### CODE REVIEW

**Review by javierm — COMMENTED** [review]
@raoulkramer Thanks for this pull request! I've left a couple of questions.

File: scripts/entrypoint.sh [hunk 1]
```diff
@@ -13,5 +13,13 @@ usermod -g "$USER_GID" consul 2> /dev/null
 chown -R -h "$USER_UID" "$BUNDLE_PATH"
 chgrp -R -h "$USER_GID" "$BUNDLE_PATH"
 
+# Remove a potentially pre-existing server.pid for Rails.
+set -e
```
Comment (javierm) [inline scripts/entrypoint.sh:17]: Just one more thing (as Columbo used to say :smile:): Is this line necessary? :thinking:

**Comment by raoulkramer** [comment]
mmm :thinking: `set -e` causes the shell to exit if any subcommand or pipeline returns a non-zero status.... I took it over from the docker docs concerning the entrypoint.sh file ... 

`rm -f /var/www/consul/tmp/pids/server.pid` is used **without** `set -e`

`if [ -f tmp/pids/server.pid ]; then rm tmp/pids/server.pid fi` is used **with** `set -e`

So I am not sure ... need to test it . :test_tube:

**Review by javierm — APPROVED** [review]

**Comment by javierm** [comment]
OK, I've approved the pull request. If your tests go well, say so and we'll merge it :ok_hand:.

**Comment by raoulkramer** [comment]
> mmm thinking `set -e` causes the shell to exit if any subcommand or pipeline returns a non-zero status.... I took it over from the docker docs concerning the entrypoint.sh file ...
> 
> `rm -f /var/www/consul/tmp/pids/server.pid` is used **without** `set -e`
> 
> `if [ -f tmp/pids/server.pid ]; then rm tmp/pids/server.pid fi` is used **with** `set -e`
> 

@javierm I finished testing. It looks that we need the `set -e` rake will be aborted without it. :boom:

**Pull request merged**